targets/bcm27xx-bcm2711

```diff
@@ -0,0 +1,10 @@
+include 'bcm27xx.inc'
+
+device('raspberrypi-4-model-b', 'rpi-4', {
+	manifest_aliases = {
```

Comment:
Let's add a comment here similar to the other RPi targets, so we know we can drop them when versions before 2023.2 are EOL. The commit message should mention that the aliases are added only for upgrades from unreleased Gluon builds.